### ISSUE TITLE
Bad SQL syntax via ejabberdctl export2odbc command - missing semicolon in some modules

### ISSUE THREAD

**Issue by MurzNN** [processone/ejabberd#1508]
I moving database of working ejabberd 16.09 server from mnesia to mysql format.
So at first - I export mnesia database to file via command
```
ejabberdctl export2sql my.jabberdomain.com /tmp/db_dump.sql
```
so file created normally.

But I got the errors on import to mysql operation, seems that this is syntax errors - missing semicolon at end of insert query, here is example:
```
insert into privacy_list(username, name, id) values ('user1', 'i-am-visible-list', 35);
delete from privacy_list_data where id=35;
insert into privacy_list_data(id, t, value, action, ord, match_all, match_iq, match_message, match_presence_in, match_presence_out) values (35, 'j', 'user1@my.jabberdomain.com', 'a', 0, 1, 0, 0, 0, 0)
insert into privacy_list_data(id, t, value, action, ord, match_all, match_iq, match_message, match_presence_in, match_presence_out) values (35, 'n', '', 'a', 1, 1, 0, 0, 0, 0)
delete from privacy_list where username='user2' and name='invisible';
insert into privacy_list(username, name, id) values ('user2', 'invisible', 36);
```
- here is missed semicolon after both `insert into privacy_list_data` queries. I change this code manually by hands to
```
insert into privacy_list(username, name, id) values ('user1', 'i-am-visible-list', 35);
delete from privacy_list_data where id=35;
insert into privacy_list_data(id, t, value, action, ord, match_all, match_iq, match_message, match_presence_in, match_presence_out) values (35, 'j', 'user1@my.jabberdomain.com', 'a', 0, 1, 0, 0, 0, 0);
insert into privacy_list_data(id, t, value, action, ord, match_all, match_iq, match_message, match_presence_in, match_presence_out) values (35, 'n', '', 'a', 1, 1, 0, 0, 0, 0);
delete from privacy_list where username='user2' and name='invisible';
insert into privacy_list(username, name, id) values ('user2', 'invisible', 36);
```
and it imports normally.

Is this bug in ejabberd or in some module? How to fix this problem without "hands" magic?

**Comment by MurzNN** [comment]
Same problem is in rostergroups table:
```
-- 
-- Mnesia table: roster
--
delete from rosterusers where username='user1' and jid='user2@office.qseo.ru';
insert into rosterusers( username, jid, nick, subscription, ask, askmessage, server, subscribe, type) values ('mosaleva', 'user2@my.jabberdomain.com', 'Ivan Ivanov', 'B', 'N', '', 'N', '', 'item');
delete from rostergroups where username='user1' and jid='user2@my.jabberdomain.com';
insert into rostergroups(username, jid, grp) values ('user1', 'user2@my.jabberdomain.com', 'Group1')
delete from rosterusers where username='user3' and jid='user4@my.jabberdomain.com';
```
missed semicolon on `insert into rostergroups` query.

**Comment by badlop** [comment]
The problem reported for privacy_list_data was fixed in 149cc9654f17c8edfbcd79a2d1973721a70eafe3.
The problem reported for rostergroups table, I can't reproduce it, maybe it was also fixed by that time.
